nereid/nereid/src/land_surface/loading.py

```diff
@@ -3,13 +3,18 @@
 
 import pandas
 
-from nereid.core.units import ureg
+from nereid.core.utils import safe_divide
+from nereid.core.units import Constants
 
 # TODO: implement fallback to e.g., "TRANS-B-10" rather than "401070-TRANS-B-10"
 # in case ref data is missing.
 
 
 def clean_land_surface_dataframe(df: pandas.DataFrame) -> pandas.DataFrame:
```

Comment:
did you mean this function: `detailed_loading_results`? the one you tagged expects any df with an 'imp_pct' column and an 'area_acres' column, so the answer there would be a bit boring.

the better story is that the function you tagged is part 1 in the detailed loading results pattern, which requires a df of your land surfaces request object, which are a list of these:
```
class LandSurface(BaseModel):
    node_id: str
    surface_key: str = Field(..., example="104506-RESSFH-B-5")
    area_acres: float
    imp_area_acres: float
```
joined to your project reference data, which are these (for the default data land_surface_data.json):
```
  "data":[
    {
      "surface_id":"10101000-COMM-A-0",
      "perv_ro_depth_inches":2.3145148876,
      "imp_ro_depth_inches":11.2305913657,
      "perv_ro_coeff":0.1915657302,
      "imp_ro_coeff":0.929523697
    },
```

plus whatever the creator of the config file joined in the `preprocess` sections....

in short, i've separated these three functions into pieces so that the structure of the data they handle is evident by the <20 line bodies of each. each function requires a dataframe with the keys accessed inside the function, and if it fails it's *most likely* because of a bad config file that should be debugged on project initialization.

at any rate, how would we show an example df in the docstring? are the required fields not enough to indicate the structure of the flat table that we are passing around?